### ISSUE TITLE
why logits of model outputs is None

### ISSUE THREAD

**Issue by Yangruipis** [linkedin/Liger-Kernel#314]
### 🐛 Describe the bug

logits are not calculated here:

https://github.com/linkedin/Liger-Kernel/blob/6ab3b9febc29f5045e6d2e27ba6bacaa4f041d91/src/liger_kernel/transformers/model/qwen2.py#L99-L110

so is it set to None for some reasons? or just a bug?
thanks?

### Versions

```
Environment Report:
-------------------
Operating System: Linux-3.10.0-1160.el7.x86_64-x86_64-with-glibc2.35
Python version: 3.10.12
PyTorch version: 2.4.0a0+f70bd71a48.nv24.06
CUDA version: 12.5
Triton version: 3.0.0
Transformers version: 4.45.0
```

**Comment by Tcc0403** [comment]
https://github.com/linkedin/Liger-Kernel/blob/6ab3b9febc29f5045e6d2e27ba6bacaa4f041d91/src/liger_kernel/transformers/monkey_patch.py#L502-L510
logits are not materialized while training with fused_linear_cross_entropy to reduce peak memory usage

https://github.com/linkedin/Liger-Kernel/blob/6ab3b9febc29f5045e6d2e27ba6bacaa4f041d91/src/liger_kernel/ops/fused_linear_cross_entropy.py#L55-L61

they are computed chunk by chunk as intermediate values only in above function

**Comment by Yangruipis** [comment]
thanks for your explain!
